src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -582,4 +586,53 @@ public StudyController(StudyService studyService, ReportService reportService) {
         Mono<List<String>> libraries = studyService.getAvailableSvgComponentLibraries();
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(libraries);
     }
+
+    @PutMapping(value = "/tree/createNode/{id}")
```

Comment:
If ids are generated by the Server, this should be a post

Reply:
done